src/Handler/StreamHandler.php

```diff
@@ -551,25 +549,24 @@ static function () use ($ident, $value, $map, $args): void {
         );
     }
 
-    private function addNotification(array &$params, callable $notify): void
+    private static function addNotification(array &$params, callable $notify): void
```

Comment:
what's the benefit of having them static? is this for some perf. reasons?

Reply:
Yeh, I just noticed these private functions could be made static. This also makes them clearer to the programmer that they won't have any side effects on the instance.